pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019,2022 Tigera, Inc. All rights reserved.
+// Copyright (c) 2019,2023 Tigera, Inc. All rights reserved.
```

Comment:
Do we need a network policy ingress rule for prometheus in here?

Reply:
Kube-controllers does not have default deny hence prometheus pod is able to fetch kubecontroller metrics once we added  egress rule in prometheus ns alone.  Please suggest, do  we need to add explicit ingress rule in kube-controllers?

Reply:
@pasanw @caseydavenport would you know more about that?

Reply:
Vara's analysis is correct in that, given a default configuration, there is no policy selecting ingress traffic to kube-controllers, so an explicit allow is not required.

However, it's likely best practice for us to ensure that any required ingress flows are explicitly allowed so that our product continues to function as expected regardless of the customers policy configuration. As I understand it, that's the main objective of allow-tigera.

If we were to explicitly allow the prometheus ingress, this would cause policy selection of ingress traffic to kube-controllers and therefore begin denying any ingress not explicitly allowed. This would need to be considered/accommodated

Reply:
^ Is that fair assessment @caseydavenport?